<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_creeracte</class>
 <widget class="QDialog" name="dlg_creeracte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="TypeActegroupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>353</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>365</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>Type d'acte</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QRadioButton" name="CCAMradioButton">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>CCAM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="AssocradioButton">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Association de codes CCAM ou  autres types d'acte</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="HorsCCAMradioButton">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Acte hors nomenclature</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>124</y>
     <width>365</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>365</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>365</width>
     <height>116</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="CodeActelabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Code</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="UpLineEdit" name="CodeActeupLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="TarifOPTAMlabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Tarif conventionnel OPTAM</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="UpLineEdit" name="TarifOPTAMupLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="TarifNonOPTAMlabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Tarif conventionnel non OPTAM</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="UpLineEdit" name="TarifNonOPTAMupLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="TarifPratiquelabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Tarif pratiqu√©</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="UpLineEdit" name="TarifPratiqueupLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpLineEdit</class>
   <extends>QLineEdit</extends>
   <header>uplineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
